README.md

```diff
@@ -44,185 +42,113 @@ _✨ A modern Anki [custom scheduling](https://faqs.ankiweb.net/the-2021-schedul
+The scheduler replaces Anki’s built in scheduler. You can find the code here: https://github.com/open-spaced-repetition/fsrs4anki/blob/main/fsrs4anki_scheduler.js
 
-The optimizer applies *Maximum Likelihood Estimation* and *Backpropagation Through Time* to estimate the stability of memory and learn the laws of memory from time-series review logs. Then, it can find the optimal retention to minimize the repetitions via the stochastic shortest path algorithm.
+The optimizer finds parameters that provide the best fit to your review history: https://colab.research.google.com/github/open-spaced-repetition/fsrs4anki/blob/v4.5.3/fsrs4anki_optimizer.ipynb
 
-For more detail on the mechanism of the FSRS algorithm, please see my papers: [A Stochastic Shortest Path Algorithm for Optimizing Spaced Repetition Scheduling (free access)](https://www.maimemo.com/paper/) and [Optimizing Spaced Repetition Schedule by Capturing the Dynamics of Memory (submit request)](https://www.researchgate.net/publication/369045947_Optimizing_Spaced_Repetition_Schedule_by_Capturing_the_Dynamics_of_Memory).
+The add-on has many useful features, you can read about it here: https://github.com/open-spaced-repetition/fsrs4anki-helper
 
-[FSRS4Anki Helper](https://github.com/open-spaced-repetition/fsrs4anki-helper) is an Anki add-on that supports the FSRS4Anki Scheduler. It has six features:
-1. **Reschedule** cards based on their entire review histories.
-2. **Postpone** due cards whose retention is higher than your target.
-3. **Advance** undue cards whose retention is lower than your target.
-4. **Balance** the load during rescheduling.
-5. **No Anki** on Free Days (such as weekends).
-6. **Disperse** Siblings (cards with the same note) to avoid interference & reminder.
+For more detail on the mechanism of the FSRS algorithm, please see my papers: [A Stochastic Shortest Path Algorithm for Optimizing Spaced Repetition Scheduling (free access)](https://www.maimemo.com/paper/) and [Optimizing Spaced Repetition Schedule by Capturing the Dynamics of Memory (submit request)](https://www.researchgate.net/publication/369045947_Optimizing_Spaced_Repetition_Schedule_by_Capturing_the_Dynamics_of_Memory).
 
-# Tutorial
+# Installation
 
 中文版请见：[FSRS4Anki 使用指北](https://zhuanlan.zhihu.com/p/636564830)
 
-## 1 Quick Start
 
-### 1.1 Enable Anki's V3 Scheduler
+## 1. Enable the V3 scheduler in Anki
 
-Preferences > Review > Enable V3 Scheduler
+Go to Tools > Preferences > Review > Enable V3 Scheduler.
 
-![image](https://github.com/open-spaced-repetition/fsrs4anki/assets/32575846/8f91fba8-9b8b-405c-8aa9-42123ba5faeb)
+![1](https://github.com/Expertium/fsrs4anki/assets/83031600/ecf7d77e-1244-4a0a-8cfd-22a5b76e0fe1)
 
-### 1.2 Paste FSRS Scheduler Code
+## 2. Export your deck or collection
```

Comment:
Should we mention the minimum limit of reviews required by the optimizer? If the user doesn't have enough data, they should skip these steps.